src/features/default-plugins.js

```diff
@@ -1,19 +1,6 @@
 import { addPlugin } from './plugins';
 
-if (typeof navigator != 'undefined' && navigator.product == 'ReactNative') {
-  // I'm in react-native
-  // Parse '9' to 9. React native uses numbers and not strings for styles.
-  addPlugin('parseKeyValue', (k, v) => {
-    const intRegex = /^([+-]?[1-9]\d*|0)$/;
-    const floatRegex = /^[-+]?[0-9]*\.?[0-9]+([eE][-+]?[0-9]+)?$/;
-    if (intRegex.test(v)) {
-      return [k, parseInt(v, 10)];
-    } else if (floatRegex.test(v)) {
-      return [k, parseFloat(v)];
-    }
-  });
-} else {
-  // I'm on the web or node js!
+if (navigator.product != 'ReactNative') {
```

Comment:
Did you test this in SSR? Assuming this will crash SSR rendering. you need the typeof navigator != 'undefined' first